doc/en/user/source/security/csp.rst

```diff
@@ -6,39 +6,51 @@ Content Security Policy
 The :guilabel:`Content Security Policy` page controls how GeoServer prepares the Content-Security-Policy
 (CSP) HTTP response headers, used to mitigate cross-site scripting and clickjacking attacks.
 
+
 Default Configuration
 ---------------------
 
 The default CSP configuration is intended to support many GeoServer use cases and allow users to
 securely run GeoServer without having to modify the configuration. It may be updated in future
 releases to fix bugs, support new features or enhance security.
 
-The default header value for most GeoServer requests will be::
-
-    base-uri 'self'; form-action 'self'; default-src 'none'; child-src 'self'; connect-src 'self'; font-src 'self'; img-src 'self' data:; style-src 'self' 'unsafe-inline'; script-src 'self';, frame-ancestors 'self';
-
-and the ``'unsafe-inline'`` and ``'unsafe-eval'`` sources will be added to the ``script-src``
-directive only for specific requests that may require unsafe JavaScript. The default configuration
-may be updated as necessary by the GeoServer developers.
+  The default header value for most GeoServer requests will be::
+  
+     base-uri 'self';
+     form-action 'self';
+     default-src 'none';
+     child-src 'self';
+     connect-src 'self';
+     font-src 'self';
+     img-src 'self' data:;
+     style-src 'self' 'unsafe-inline';
+     script-src 'self';
+     frame-ancestors 'self';
```

Comment:
I have broken this into individual lines for readability; you can now scan the page and see what is changing.

The one thing that looks like a mistake was the `;,` below:
```
script-src 'self';, frame-ancestors 'self'
```